src/controller/handler/handlers.factory.ts

```diff
@@ -27,6 +28,8 @@ export class HandlersFactory {
                 return new IssueCredential(agentGovernance, http, cache);
             case 'problem_report':
                 return new ProblemReport(agentGovernance, http, cache);
+            case 'basicmessage':
```

Comment:
To match the other cases, could you make this `basic_message`?

Reply:
no I cannot.  'basic_message' is the value sent to us from acapy.

Reply:
Oh, my bad, I thought these were under our control.